lib/amorail/entity/finders.rb

```diff
@@ -52,7 +52,7 @@ def load_record(id)
         remote_url('list'),
         id: id
       )
-      handle_response(response, 'load')
+      handle_response(response, 'load') || nil
```

Comment:
Why not returning `nil` from `handle_response` instead? `... || nil` looks a little bit awkward